### ISSUE TITLE
mousewheel scrolling in dialogs doesn't change the moueover state of widgets

### ISSUE THREAD

**Issue by gfgtdf** [wesnoth/wesnoth#1795]
You can observe this for example in the mp lobby game list:
When you have the mouse over one list item then scroll the list via mousewheel the old item still have the mouseover state. If you now click the mouse (which is now over a sifferent list item) without moving it first, the old list item will stay in it's mouseover state untill you move the mouse over this item again.

**Comment by Wedge009** [comment]
This sounds very similar to, if not the same as, #1335.

**Comment by gfgtdf** [comment]
That one is themewml this one is gui2. Also the reported behaviour is different

**Comment by Wedge009** [comment]
Okay, but it still seems to be another case of scroll [something] and clicking without moving the mouse results in unexpected behaviour.

**Comment by soliton-** [comment]
I agree that the issue is in principle the same that on mouse wheel scroll the highlighting is not updated. Btw, #1335 was just fixed by accepting clicks even if not active/highlighted. In that case as well updating the highlighting/active status would still be good. (Though perhaps it will be converted to GUI2 before...)
